docs/detections/alert-suppression.asciidoc

```diff
@@ -33,6 +33,10 @@ For example, if a rule runs every 5 minutes but you don't need alerts that frequ
 image::images/alert-suppression-options.png[Alert suppression options,400]
 --
 
+. Under *If a suppression field is missing*, choose how to handle events with missing suppression fields (one or more of the *Suppress alerts by* fields doesn't exist on an event):
```

Comment:
I'd like to keep the original here, because it intentionally explains what "missing fields" means in a different way than just saying "fields are missing."

Reply:
@joepeeples I like this one you suggested here ⬆️